tests/acceptance/organization-test.js

```diff
@@ -20,23 +20,24 @@ test("it displays the organization's details", (assert) => {
   let sluggedRoute = server.schema.sluggedRoutes.create({
     slug: 'test_organization',
   });
+
   let organization = sluggedRoute.createOwner({
     name: 'Test Organization',
     slug: 'test_organization',
     description: 'Test organization description.'
   }, 'Organization');
+
   sluggedRoute.save();
 
   for (let i = 1; i <= 3; i++) {
-    organization.createProject({
-      slug: `test_project_${i}`,
-      title: `Test project ${i}`
-    });
+    server.schema.projects.create({ organization: organization });
```

Comment:
Just an FYI, for future needs, `server.schema.modelName.create` doesn't use factories, while `server.create('model-name, attrs)` does, in case we want some data to be auto-generated.